api/src/main/java/org/pmiops/workbench/google/CloudStorageServiceImpl.java

```diff
@@ -35,6 +35,14 @@ public String readBlockscoreApiKey() {
     return readToString(getCredentialsBucketName(), "blockscore-api-key.txt").trim();
   }
 
+  public String readMailChimpApiKey() {
```

Comment:
Seems like it may be easier to manage if we had one file that had both the API key and list ID?

Reply:
Yeah, that’s probably true.

Reply:
Nit: I think you could have MailChimpService cache both of these... we're not going to change API keys or  list IDs once we get things working.